app/server.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask, render_template, request, url_for, abort, Response, make_response
-from scrapers import feedgen
+from scraper import feedgen
```

Comment:
The folder name is `scrapers` right? And feedgen has been moved to __init__ as of #141

Reply:
Scrapers file is showing error so I used the scraper.py file to fetch the data.

Reply:
We are moving the scrapers, so I think you should use that module only.

May be try to fix the errors?